tendrl/alerting/api/manager.py

```diff
@@ -0,0 +1,114 @@
+from flask import Flask
+from flask import request
+from flask import Response
+import logging
+from multiprocessing import Event
+from multiprocessing import Process
+from tendrl.alerting.etcd.exceptions import EtcdError
+from tendrl.alerting.etcd.manager import EtcdManager
+from tendrl.alerting.exceptions import AlertingError
+from tendrl.alerting.exceptions import InvalidRequest
+from tendrl.alerting.notification.exceptions import InvalidHandlerConfig
+from tendrl.alerting.notification.exceptions import NotificationPluginError
+from tendrl.alerting.notification.manager import NotificationPluginManager
+from tendrl.common.config import TendrlConfig
+
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+app = Flask(__name__)
+
+
+@app.route("/alerting/notification_medium/supported")
+def get_handlers():
+    try:
+        return Response(
+            NotificationPluginManager().get_handlers(),
+            mimetype='application/json'
+        )
+    except (
+        AlertingError,
+        NotificationPluginError
+    ) as ex:
+        raise ex
```

Comment:
I have observed InvocationTargetException appearing for any exception thrown from these handlers. I'll find a way to appropriately set error code and message instead of the default InvocationTargetException kind of error